client/src/serviceWorker.ts

```diff
@@ -0,0 +1,145 @@
+// This optional code is used to register a service worker.
```

Comment:
Out of curiousity. Is that generated or something we build. For angular this was generated dynamically based on browserify build

Reply:
That's the create react app service worker, so it's all generated by them. I don't believe it's dynamic

Reply:
It's generated by create react app